### PR TITLE
fix: sqllab schema select error msg

### PR DESCRIPTION
**Pull request by pkdotson**
### SUMMARY
This pr fixes and issue where if the sqllab backend persistence flag is on the param being passed into the dispatch function is not always a object and will subsequently caused the call to fail.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
before
Screen Shot 2022-02-02 at 9 33 35 PM

after

https://user-images.githubusercontent.com/17326228/152286717-1aa52a70-643a-428e-b5af-f9715383b57e.mov

### TESTING INSTRUCTIONS
Go to sql lab and click the schema and database selects. The error toast should not show.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/SqlLab/actions/sqlLab.js [hunk 1]
```diff
@@ -775,8 +775,10 @@ export function queryEditorSetDb(queryEditor, dbId) {
 }
 
 export function queryEditorSetSchema(queryEditor, schema) {
+  console.log('queryEditor', typeof queryEditor, '---schema -->', schema)
```
Comment (villebro) [inline superset-frontend/src/SqlLab/actions/sqlLab.js:778]:
```suggestion
  console.log('queryEditor', typeof queryEditor, '---schema -->', schema)
```
Oops!

File: superset-frontend/src/SqlLab/actions/sqlLab.js [hunk 2]
```diff
@@ -775,8 +775,10 @@ export function queryEditorSetDb(queryEditor, dbId) {
...
   return function (dispatch) {
-    const sync = isFeatureEnabled(FeatureFlag.SQLLAB_BACKEND_PERSISTENCE)
+    console.log('featureflag *******', isFeatureEnabled(FeatureFlag.SQLLAB_BACKEND_PERSISTENCE))
```
Comment (villebro) [inline superset-frontend/src/SqlLab/actions/sqlLab.js:780]: Oops pt 2!

**Review by villebro — COMMENTED** [review]
Thanks for working on this, I just bumped into this the other day!

**Review by lyndsiWilliams — APPROVED** [review]

**Comment by rosemarie-chiu** [comment]
🏷 preset:2022.5

**Pull request merged**